.github/workflows/main.yml

```
@@ -34,111 +34,34 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node }}
-      - run: yarn
+      - run: npm install
       - run: npx nyc --reporter=lcov aegir test -t node -- --bail
```

nyc isn't necessary any more

```suggestion
      - run: npx aegir test -t node --cov --bail
```